app/src/main/java/com/taskruler/MainActivity.kt

```diff
@@ -19,7 +19,7 @@ import org.koin.androidx.viewmodel.ext.android.viewModel
 
 class MainActivity : ComponentActivity() {
 
-    private val viewModel: MainViewModel by viewModel<MainViewModel>()
+    private val viewModel: MainViewModel by viewModel()
```

Comment:
<MainViewModel> was an explicit type argument and could be removed/was redundant